pubnub/lib/src/dx/_endpoints/push.dart

```diff
@@ -56,9 +56,11 @@ class ListPushChannelsParams extends Parameters {
   PushGateway pushGateway;
   Environment? environment;
   String? topic;
+  String? start;
+  int? count;
 
   ListPushChannelsParams(this.keyset, this.deviceId, this.pushGateway,
-      {this.topic, this.environment});
+      {this.topic, this.environment, this.start, this.count});
```

Comment:
Nitpick, you can add a trailing comma `,` after `this.count` so your IDE should break the line